modules/eks-aws/values.tmpl.yaml

```diff
@@ -135,6 +135,35 @@ kube-prometheus-stack:
             - prometheus.apps.${cluster_name}.${base_domain}
 
   grafana:
+    grafana.ini:
+      auth.generic_oauth:
+        name: Cognito
+        enabled: true
+        allow_sign_up: true
+        client_id: ${cognito_user_pool_client_id}
+        client_secret: ${cognito_user_pool_client_secret}
+        scopes: "openid profile email"
+        auth_url: https://${cognito_user_pool_domain}.auth.${aws_default_region}.amazoncognito.com/oauth2/authorize
+        token_url: https://${cognito_user_pool_domain}.auth.${aws_default_region}.amazoncognito.com/oauth2/token
+        api_url: https://${cognito_user_pool_domain}.auth.${aws_default_region}.amazoncognito.com/oauth2/userInfo
```

Comment:
Isn't there any discovery endpoint URL configuration in Grafana to avoid declaring all endpoints?